elodie/filesystem.py

```diff
@@ -213,13 +218,38 @@ def process_file(self, _file, destination, media, **kwargs):
             shutil.move(_file, dest_path)
             os.utime(dest_path, (stat.st_atime, stat.st_mtime))
         else:
-            shutil.copy2(_file, dest_path)
+            shutil.copy(_file, dest_path)
+            self.set_date_from_filename(dest_path)
 
         db.add_hash(checksum, dest_path)
         db.update_hash_db()
 
         return dest_path
 
+    def set_date_from_filename(self, file):
```

Comment:
Okay. I didn't realize this was already present and using the existing format that we prefix file names with.

I can add a commit and open a PR against your master branch that once you merge will update this pr.